core/nodeFilesystemShared/src/files/PathsNodeJs.kt

```diff
@@ -10,6 +10,7 @@ import kotlinx.io.node.buffer
 import kotlinx.io.node.fs
 import kotlinx.io.node.path as nodeJsPath
 
+@Suppress("EXPECT_ACTUAL_CLASSIFIERS_ARE_IN_BETA_WARNING")
```

Comment:
same as above: should be not needed after https://github.com/Kotlin/kotlinx-io/pull/325 was merged